runtime-light/stdlib/server/args-functions.h

```diff
@@ -11,3 +11,13 @@ inline Optional<string> f$ini_get(const string &key) noexcept {
   const auto &component_st{ComponentState::get()};
   return component_st.ini_opts.has_key(key) ? Optional<string>{component_st.ini_opts.get_value(key)} : Optional<string>{false};
 }
+
+inline string f$get_engine_version() noexcept {
```

Comment:
How are these functions related to command line arguments? Let's have them somewhere else